src/coreclr/jit/hwintrinsiccodegenxarch.cpp

```diff
@@ -1444,6 +1448,8 @@ void CodeGen::genNonTableDrivenHWIntrinsicsJumpTableFallback(GenTreeHWIntrinsic*
 
         case NI_AVX512F_ConvertToInt32:
         case NI_AVX512F_ConvertToUInt32:
+        case NI_AVX10v1_ConvertToInt32:
+        case NI_AVX10v1_ConvertToUInt32:
```

Comment:
Similar question as to before around `NI_AVX10v1_X64_ConvertTo*`

Reply:
and `NI_AVX10v1_Fused*`

Reply:
For X64, I will wait for concluding https://github.com/dotnet/runtime/pull/101938#discussion_r1607436860
For FMA, I will wait for concluding https://github.com/dotnet/runtime/pull/101938#discussion_r1605538299

Reply:
I have added all the instructions in FMA to AVX10v1.